src/platform/packages/shared/kbn-es/src/serverless_resources/project_roles/security/roles.yml

```diff
@@ -63,6 +63,7 @@ viewer:
         - feature_visualize_v2.all
         - feature_savedQueryManagement.all
         - feature_dataQuality.all
+        - feature_agentBuilder.read
```

Comment:
Also made the changes for roles here: https://github.com/elastic/elasticsearch-controller/pull/1389